services/tenant-ui/frontend/src/main.ts

```diff
@@ -37,6 +37,22 @@ async function loadApp() {
       after((result) => {
         console.log('configuration loaded from server.');
         console.log(result);
+        // Matomo Setup
+        const MATOMO_URL: string = result?.frontend.matomoUrl;
+        if (MATOMO_URL) {
+          // @ts-ignore
```

Comment:
Tried locally and I don't see it needing this ts-ignore? Builds this part for me.

Reply:
Thanks, this was a hold over from a previous change and I forgot to remove the ts-ignore. eslint was kind enough to point it out